include/Core/Posts/Modules/DeletePostsByCommentsModule.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+namespace BulkWP\BulkDelete\Core\Posts\Modules;
+
+use BulkWP\BulkDelete\Core\Posts\PostsModule;
+
+defined( 'ABSPATH' ) || exit; // Exit if accessed directly.
+
+/**
+ * Delete Posts by Comments Module.
+ *
+ * @since 6.1.0
```

Comment:
Change this (and the version in all other places) to `6.0.0`

Reply:
Fixed.